Add comments on Phoenix section (#84)
lambdaclass/lambdaclass_hacking_learning_path: README.md

- [Phoenix Chat Example](https://github.com/dwyl/phoenix-chat-example)

**Installing Phoenix**

> - For Phoenix, all dependencies and versions needed for installing can be found [here](https://hexdocs.pm/phoenix/installation.html). Postgres is recommended as the de facto database. To install versions of erlang and elixir needed for Phoenix in Action, use asdf to install previous versions.
> - If asdf doesn't let you install previous version of erlang [this](https://github.com/asdf-vm/asdf-erlang/issues/221) might help.  

Reviewer: ```suggestion
- For Phoenix, all dependencies and versions needed for installing can be found [here](https://hexdocs.pm/phoenix/installation.html). Postgres is recommended as the de facto database. To install older versions of Erlang and Elixir, use `asdf` to install them.
- If `asdf` doesn't let you install a previous version of Erlang [this](https://github.com/asdf-vm/asdf-erlang/issues/221) might help.  
```

I think you should put that it is better to install Erlang and Elixir through asdf.
